drivers/SmartThings/matter-sensor/src/air-quality-sensor/init.lua

```diff
@@ -378,13 +378,11 @@ local conversion_tables = {
 local function unit_conversion(value, from_unit, to_unit)
   local conversion_function = conversion_tables[from_unit][to_unit]
   if conversion_function == nil then
-    log.info_with( {hub_logs = true} , string.format("Unsupported unit conversion from %s to %s", unit_strings[from_unit], unit_strings[to_unit]))
-    return 1
+    return nil, string.format("Unsupported unit conversion from %s to %s", unit_strings[from_unit], unit_strings[to_unit])
```

Comment:
Can you add a unit test for this as well?

Reply:
Done in [602bced](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1582/commits/602bced7425981a5551e35879da067bf0545b606)